spec/jekyll_seo_tag_spec.rb

```diff
@@ -196,6 +196,60 @@
       end
     end
 
+    context "with image.path, image.height, and image.width" do
```

Comment:
In other words, feels like the smell is that schema goods are being duplicated by implementation details and not behavior. Seems like a good candidate for a letter refactor.